source-browser.html

```diff
@@ -166,45 +203,129 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
               ),
             );
         });
-
+          
         $('#topics, #loading').toggleClass('display-none');
       }
-
+        
       function refreshTopics() {
         localStorage.clear();
         showTopics();
       }
-
+        
       function showSearchInput() {
         $('#search')
           .removeClass('display-none')
           .focus();
       }
+        
+
+        $(window).scroll(function() {
+            var top = $(window).scrollTop();
+            var bottom = $(window).scrollTop() + $(window).height();
+            var langtop = $("#languages").position(); 
```

Comment:
These should be adjusted via a loop that goes through the topics, not hardcoded to only work with these.